src/components/ec-summary-info/ec-summary-info.vue

```diff
@@ -0,0 +1,153 @@
+<template>
+  <div
+    class="ec-summary-info"
+    data-test="ec-summary-info"
+  >
+    <div
+      v-if="iconName"
+      class="tw-mr-12"
+    >
+      <ec-icon
+        class="ec-summary-info__main-icon"
+        :data-test="`ec-summary-info__main-icon ec-summary-info__main-icon__${iconName}`"
+        :name="iconName"
+        :size="16"
+      />
+    </div>
+
+    <slot>
+      <div
+        v-if="lineItems.length > 0"
+        class="ec-summary-info__content"
+      >
+        <div
+          data-test="ec-summary-info__content-lines"
```

Comment:
Hi can we please double check we need this. 

We have a div `ec-summary-info__content` inside a div `ec-summary-info__content-lines`

Does the parent div add any value? Can we merge those 2?

Reply:
merged them, thanks